courses/machine_learning/deepdive2/feature_engineering/solutions/Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2721 @@
+{
```

Comment:
<p>Should line 3 be "your-gcp-project-here" so line 11 can fire if the student forgets to change it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes.  Changed.</p>